cfme/fixtures/events.py

```diff
@@ -81,6 +81,24 @@ def pytest_runtest_call(item):
                                 "Event {} did not come!".format(event['event']))
             else:
                 logger.info('Seems like all events have arrived!')
+        elif "register_targeted_refresh" in item.funcargnames:
```

Comment:
So basically we need to wait some time after main code block (of the test) executes and check if specific lines have appeared in the evm.log. But do we really need to hook this here? Couldn't we just do it a fixture after the `yield`? 

@izapolsk maybe ^, since you've been involved in event listener.
@sasoc have you tried it and there was an error?

Reply:
In any case @sasoc do we really want to extend their function here? I'd rather implement our own hook function in the same file that our fixture is defined...

Reply:
@miha-plesko I will implement it in my own hook function.

Reply:
@sasoc can you please try to implement everything as a normal fixture without the hook function instead? As far as I can understand the hook function is only needed because of the soft_assert that you don't really need IMAO.

Reply:
@miha-plesko Do you have an advice what to do in case of multiple entities to check, if I use normal fixture? For example if we check for subnet, enterprise and zone. What to do in case that 2 of them don't trigger and one does. How and when to display an error?

Reply:
Concat errors in a single string and raise it with hard assert. I mean, doing all this woodoo magic just to call soft_assert sounds like a nonsense to me since we have a list of all the found matches at hand all the time.

Reply:
@miha-plesko Ok thank you for the advice.